spec/models/attorney_rich_text_auditing_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Attorney do
+  describe 'auditing rich text fields' do
+    subject(:update_notes) { attorney.update!(notes: 'Rich text content') }
```

Comment:
although I'm not sure if this is wrong, we don't typically use a let variable for this situation, I would move the `attorney.update!` code into the {} below instead of extracting as a let